resources/migrations/001_update_migrations.yaml

```diff
@@ -6055,6 +6055,19 @@ databaseChangeLog:
             constraintName: idx_cache_config_unique_model
             columnNames: model, model_id
 
+  - changeSet:
+      id: v50.2024-03-21T17:41:00
+      author: qnkhuat
+      comment: Add metabase_table.estimated_row_count
+      changes:
+        - addColumn:
+            tableName: metabase_table
+            columns:
+              - column:
+                  name: estimated_row_count
+                  type: integer
```

Comment:
This needs to be a biginteger type to match `pg_stat_user_tables.n_live_tup`. Otherwise it'll fail for tables over ~2 billion rows

Reply:
good catch! updated [8e78370](https://github.com/metabase/metabase/pull/39494/commits/8e78370cb7edb0b9846a615e587bf9c4a9034635)